index.py

```diff
@@ -682,8 +682,27 @@ def deg2num(lat_deg, lon_deg, zoom):
 
 
 if __name__ == "__main__":
-    web.wsgi.runwsgi = lambda func, addr=None: web.wsgi.runfcgi(func, addr)
+    # standalone run
+
+    # stop listening on 0.0.0.0
+    if not sys.argv[1:]:
+        sys.argv.append('127.0.0.1')
     app = web.application(urls, globals())
-    app.run()  # standalone run
+
+    # serve all files as static
+    def root_static(app):
```

Comment:
why is it needed now and wasn't needed before?

Reply:
Because when `web.py` is run in standalone mode without web server, it assumes that all static resources are located in `./static` dir.

Reply:
@abitrolly ,

I like idea to server static for local dev, but I not sure that middleware the best solution there, for example it looks you can return data from other than `static` directories.
What do you think about a bit other approach: add appropriate handler: to `urls` in this section?

Reply:
Yes, the middleware returns any file if it exists. The cleanest way would be to move all assets in `./static` subdir, which is supported by many frameworks for development, [including `web.py`](https://webpy.org/cookbook/staticfiles).

I didn't go with custom handler for two reasons - 1. I didn't want to include the route that is only used for development, 2. I wanted to reuse `web.py` code that is better tested for security.

Reply:
@abitrolly , I don't say about apache or nginx config now, I say about web.py reusing there, for example adding handler there https://github.com/osm-by/OpenStreetMap.by/blob/master/index.py#L46 (doc https://webpy.org/cookbook/url_handling) for my perspective can looks a bit cleaner and still reuse `web.py`.

Reply:
@tbicr right, but then you need to write the static serving handler from scratch. And it must be good enough in preventing all kinds of escaping tricks to gain access to files outside of repository root. Something that `web.py` and Apache already took care of.

It is possible to refactor `web.py` static wsgi into the static handler, but it will take some time.